### PR TITLE
Update build.yml

### LINKED ISSUE #13: Build and deploy not kicking off
Build and deploy not kicking off

### PR DESCRIPTION
**Pull request by Zeroto521**
fix https://github.com/Flow-Launcher/flow-launcher.github.io/issues/13

### CODE REVIEW

**Comment by Zeroto521** [comment]
`Enable auto-merge` and no the pr trigger cause this problem.

There was only the push trigger before.
I clicked the `Enable auto-merge` button, then it would no `push` likely after `merge`, only `merge`.

**Review by jjw24 — APPROVED** [review]

**Comment by jjw24** [comment]
Makes sense, thank you

**Comment by Zeroto521** [comment]
No thanks.

Now it needs some new commits to activate the `build & deploy` gh action.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -1,33 +1,44 @@
 name: Build & Deploy
+
+
 on:
   push:
     branches:
-      - master
```
Comment (jjw24) [inline .github/workflows/build.yml:5]: Just realized this should be master not main correct? @Zeroto521

Comment (Zeroto521) [reply .github/workflows/build.yml:5]: I think it better to let one branch manage to build and deploy, otherwise different versions would get conflict.

Comment (Zeroto521) [reply .github/workflows/build.yml:5]: https://github.com/Flow-Launcher/flow-launcher.github.io/commit/f86526fe3b54a4c7c2913ab5b4795897c42e78ae#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721

You are right, I forget that

**Pull request merged**